src/pathfinding_service/model/token_network.py

```diff
@@ -140,6 +141,32 @@ def _calculate_fees(self) -> Optional[List[FeeAmount]]:
 
         return fees
 
+    def _get_matrix_users(self) -> Optional[Dict[str, str]]:
+        # Check node reachabilities
+        user_ids: Dict[str, str] = {}
+        for node in self.nodes:
+            node_user_ids = self.reachability_state.get_userids_for_address(node)
+            node_user_id = None
+            for user_id in node_user_ids:
+                if self.reachability_state.get_userid_presence(user_id) in [
+                    UserPresence.ONLINE,
+                    UserPresence.UNAVAILABLE,
+                ]:
+                    node_user_id = user_id
```

Comment:
You could directly add to `user_ids` here instead of in line 167.
```suggestion
                    user_ids[to_checksum_address(node)] = node_user_id
```

Reply:
I rearranged the function a bit, please have a look